src/notifications/tests/test_models.py

```diff
@@ -1,3 +1,5 @@
+import unittest
```

Comment:
from unittest import TestCase.  No need to import the entire module.

Reply:
done